### PR TITLE
Enable copy paste for datatables

### LINKED ISSUE #7762: Copy & paste from datatable
I don't believe it is possible already, but could you add the ability to copy & paste from the datatable.
My use case would be for copying and pasting between Bokeh and Excel.
Many thanks!

### PR DESCRIPTION
**Pull request by alhoo**
Use slickgrid external copy paste manager to add copy paste support for bokeh datatables.

- [x] issues: fixes #7762
- [x] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

**Comment by bryevdv** [comment]
This worked well with local testing and has good new tests. I can't see any reason not to merge. 

cc @mattpap the 6pac repo seems at least moderately active, I'd suggest we propose upstream changes to get rid of the need for our fork

File: bokehjs/package-lock.json [hunk 1]
```diff
@@ -1933,6 +1933,43 @@
       "integrity": "sha1-m387DeMr543CQBsXVzzK8Pb1nZQ=",
       "dev": true
     },
+    "patch-package": {
```
Comment (mattpap) [inline bokehjs/package-lock.json]: Where this come from? It isn't in `package.json`.

Comment (mattpap) [reply bokehjs/package-lock.json]: `npm install` needs to be run and changes to this file commited.

**Comment by mattpap** [comment]
> I'd suggest we propose upstream changes to get rid of the need for our fork

We can try, but that may be a hard sale given that we use completely different module and packaging scheme.

File: bokehjs/package-lock.json (hunk 1 above)
Comment (bryevdv) [reply bokehjs/package-lock.json]: ah missed that, it's cruft from the other version of the PR. I will fix it up in another small PR

**Pull request merged**